configuration/requests.go

```diff
@@ -33,12 +33,13 @@ type SubscribeRequest struct {
 	Metadata map[string]string `json:"metadata"`
 }
 
-// UnSubscribeRequest is the object describing a request to unsubscribe configuration.
-type UnSubscribeRequest struct {
-	Keys []string `json:"keys"`
```

Comment:
Does it means we can not use one API to unsubscribe many config?

Reply:
One unsubscribe call is only for stopping only on subscription created before. The previous subscription may contains many keys, which means that we can unsubscribe many config once. @daixiang0

Reply:
So it is a breaking change for this API?

Reply:
Yes, redis is the first impl of this API. @LaurenceLiZhixin